spec/rails_helper.rb

```diff
@@ -35,8 +35,10 @@
 end
 RSpec.configure do |config|
   config.include Rails.application.routes.url_helpers
-  # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
-  config.fixture_path = Rails.root.join('spec', 'fixtures').to_s
+
```

Comment:
@bess I wonder if this is really needed.  I didn't have this change in describe.  I think spec/fixtures is the default.  Not that being specific is necessarily bad...

Reply:
Convo had in slack https://pulibrary.slack.com/archives/C02PHG9MGU8/p1727444106734649?thread_ts=1727441984.978499&cid=C02PHG9MGU8 This IS needed